.gitlab-ci.yml

```diff
@@ -1,6 +1,7 @@
 ---
 include:
   - .gitlab/.pre/cancel-prev-pipelines.yml
+  - .gitlab/.pre/linters.yml
```

Comment:
💬 for a following PR
We might want to be more precise and split the linters in 2 categories, the "traditional" language linters (shell, python, go) from the additional ones (licenses, copyrights, gitlab (hidden in `test_gitlab_configuration`).
My suggestion:
- extract the lint jobs from `source_test` and create a `language_linters.yml` (and associated stage) with lint shell, go & python
- put all other linters in a `test_pr_configuration` that can stay in the `.pre` stage (which is supposed to be a "[before build](https://docs.gitlab.com/ee/ci/yaml/#stages)" stage)

wdyt?

Reply:
I like the before/after build split depending on what it lints, I won't do this in this PR though !